docs/_src/usage/usage/reader.md

```diff
@@ -263,17 +263,22 @@ print_answers(prediction, details="all")
                        'She travels with her father, Eddard, to '
                        "King's Landing when he is made Hand of the "
                        'King. Before she leaves,',
-            'probability': 0.9899835586547852,
+            'confidence': 0.9899835586547852,
             ...
         },
     ]
 }
 ```
 
 In order to align this probability score with the model's accuracy, finetuning needs to be performed
-on a specific dataset. Have a look at this [FARM tutorial](https://github.com/deepset-ai/FARM/blob/master/examples/question_answering_confidence.py)
-to see how this is done. 
-Note that a finetuned confidence score is specific to the domain that its finetuned on. 
+on a specific dataset. 
+To this end, the reader has a `calibrate_confidence_scores()` method. 
+This method needs the has the same input parameters as the `eval()` method because the calibration of confidence scores is performed on a dataset that comes with gold labels.
```

Comment:
Typo: "needs the has the.."

"has the same input parameters as the `eval()` method" => Let's avoid that users start here to search for the eval method to understand what params this one expects. Let's rather put all information needed directly here or linking the API reference for calibrate_confidence_scores(). I guess you are already trying to say that documentstore is the only major parameter needed here, but I wouldn't really understand it that way by just reading the text.